extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/JDBCEnvironment.java

```diff
@@ -271,5 +271,29 @@ public boolean shouldUseBatchExecutor() {
         return true;
 
     }
+    
+    /**
+     * Returns a boolean value that indicates whether or not usernames should
+     * be treated as case-sensitive.
+     * 
+     * @return
+     *     true if usernames should be treated as case-sensitive, or false if
```

Comment:
I noticed that `@return` annotation in `ConfigurationService.java` capitalizes the initial "True", whereas this doesn't. I'm ok with either, but it we should probably try to be consistent.

Reply:
I think I've made this standard/consistent across all of the changes :-).